toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -414,6 +427,23 @@ func (r *RpmRepoCloner) Clone(cloneDeps bool, packagesToClone ...*pkgjson.Packag
 		}
 	}
 
+	for _, packageNameToClone := range packageNamesToClone {
+		logger.Log.Debugf("Cloning raw name (%s).", packageNameToClone)
+
+		finalArgs := append(constantArgs, packageNameToClone)
+		err = r.chroot.Run(func() (chrootErr error) {
+			prebuilt, chrootErr := r.clonePackage(finalArgs)
+			if !prebuilt {
+				allPackagesPrebuilt = false
+			}
+			return
+		})
+
+		if err != nil {
+			return
+		}
+	}
```

Comment:
I think the second `for` loop and the `packagesToClone` argument can be dropped. What do you think about making `Clone` convert the `packagesToClone` list to a `packageNamesToClone` list by calling `convertPackageVersionToTdnfArg` and then passing that new list into `CloneRawPackageNames`? The `CloneRawPackageNames` method would do what `cloneInternal` is currently doing (minus the additional `for` loop).

Also, to make the distinction clearer, we could rename `Clone` to `ClonePackageVers` or something along these lines, but that's a minor detail and I'm not married to this idea.

Reply:
Seems like a reasonable approach, done.